frontend/src/vault/deposit/DepositForm/chainOptionsConfig.ts

```diff
@@ -81,6 +81,77 @@ export const requiredFieldsPerChainAction = {
           ),
       }),
   },
+  bond_with_lp: {
+    fields: [
```

Comment:
when bonding with lp units , it need to have an asset name , like the following
`BOND:ASSET:LPUNITS:NODEADDRESS`
you can see what assets are bondable now here: https://mayanode.mayachain.info/mayachain/pools `"bondable": true`